testing/static-ip-conflict-with-pool.yaml

```diff
@@ -0,0 +1,80 @@
+# Environment file demonstrating how to pre-assign IPs to all node types
```

Comment:
Oh, it's just for manual testing to make sure the validator does actually find the static ip errors like it's supposed to.

There's another file already in there that should be testing the rest of the functionality.